js/content/community.js

```diff
@@ -858,6 +878,59 @@ let ProfileHomePageClass = (function(){
     return ProfileHomePageClass;
 })();
 
+
+let GroupHomePageClass = (function(){
+
+    function GroupHomePageClass() {
+        this.addExtraLinks();
+    }
+
+    GroupHomePageClass.prototype.addExtraLinks = function() {
+
+        let groupId = GroupID.getGroupId();
+        let iconType = "none";
+        let images = SyncedStorage.get("show_profile_link_images");
+        if (images !== false) {
+            iconType = images === "color" ? "color" : "gray";
+        }
+
+        let links = [
+            {
+                "id": "steamgifts",
+                "link": `//www.steamgifts.com/go/group/${groupId}`,
+                "name": "SteamGifts",
+            }
+        ];
+
+
+        // Build the links HTML
+        let htmlstr = "";
+
+        links.forEach(link => {
+            if (!SyncedStorage.get("group_" + link.id)) { return; }
+            htmlstr +=
+                `<div class="es_profile_link profile_count_link">
+                    <a class="es_sites_icons es_${link.id}_icon es_${iconType}" href="${link.link}" target="_blank">
+                        <span class="count_link_label">${link.name}</span>
+                    </a>
+                </div>`;
+
+        });
+
+        // Insert the links HMTL into the page
+        if (htmlstr) {
+            let linksNode = (document.querySelector(".responsive_hidden > .rightbox")).parentNode;
+            if (linksNode) {
+		HTML.afterEnd(linksNode,'<div class="rightbox_header"></div><div class="rightbox"><div class="content">'+htmlstr+
+					'</div></div></div><div class="rightbox_footer"></div>');
+	    }
```

Comment:
Usually for HTML strings with variables in them, template literals is used (`` `String with ${variable}` ``).
Also this bracket should be properly indented.